src/DependencyInjection/MercureExtension.php

```diff
@@ -251,6 +252,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->register(Authorization::class)
             ->addArgument(new Reference(HubRegistry::class))
+            ->addArgument($config['default_lifetime'] ?? $container->get('session.storage.options', ContainerInterface::NULL_ON_INVALID_REFERENCE)['cookie_lifetime'] ?? null)
```

Comment:
`session.storage.options` is not a service but a parameter,  right?

Reply:
Yes my bad. Fixed now using a compiler pass and the proper call to `getParameter`